lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -89,6 +89,10 @@ sub _init
       event_persister2         => main::alloc_port( "event_persister2[$idx]" ),
       event_persister2_metrics => main::alloc_port( "event_persister2[$idx].metrics" ),
       event_persister2_manhole => main::alloc_port( "event_persister2[$idx].manhole" ),
+
+      writer         => main::alloc_port( "writer[$idx]" ),
+      writer_metrics => main::alloc_port( "writer[$idx].metrics" ),
+      writer_manhole => main::alloc_port( "writer[$idx].manhole" ),
```

Comment:
I couldn't think of a better name than "writer"

Reply:
Similarly unimaginative, but simply `stream_writer`? Or `non_events_stream_writer`? I don't think just seeing "writer" in the logs would tell me much.
<sub>or `write-o-tron 6000`?</sub>